develop-docs/sdk/data-model/event-payloads/contexts.mdx

```diff
@@ -261,14 +261,17 @@ To summarize:
 
 : _Optional_. An unprocessed description string obtained by the operating system. For some well-known runtimes, Sentry will attempt to parse `name` and `version` from this string, if they are not explicitly given.
 
-`distribution`
+`distribution_name`
 
-: _Optional_. An object that provides meta-data for Linux distributions. The values correspond to entries from the [`/etc/os-release`](https://www.freedesktop.org/software/systemd/man/latest/os-release.html#Options) configuration. Contains the following keys:
```

Comment:
by flattening, this section was removed (I couldn't find a natural way to fit it in with the remaining keys). I think it still makes sense to link to the os-release documentation that was linked before.

Reply:
I added links to the relevant section on the `os-release` page for each of the remaining keys.